clang/docs/ReleaseNotes.rst

```diff
@@ -136,6 +136,9 @@ Improvements to Clang's diagnostics
 Improvements to Clang's time-trace
 ----------------------------------
 
+- Clang now adds file and line information for -ftime-trace profile as separate fields
+  (as ``event["args"]["file"]`` and ``event["args"]["line"]`` respectively).
```

Comment:
Could you mention what the previous behavior was, i.e. that it was reported as a string in `detail`. It might be read as if we are keeping the old behavior too, but it may actually be a change that breaks compatibility and it's useful to mention that.